hello-books-swagger.json

```diff
@@ -154,6 +151,40 @@
         }
       }
     },
+    "/books/request": {
+      "post": {
+        "tags": ["Users"],
+        "summary": "Handles the creation of book request by users",
+        "description": "This endpoint enables user to be able to make request for a book that is not yet in the library",
+        "consumes": ["application/json"],
+        "produces": ["application/json"],
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "description": "This is the body of the request object containing information about the book the user wants to request for.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/BookRequest"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Book request was created successfully"
+          },
+          "409": {
+            "description": "Duplicate request was made for a book"
+          },
+          "422": {
```

Comment:
Just to clarify, I went through the books controller file and didn't see an instance where 422 was returned. Please, could you confirm if you actually returned it for this feature? If you did, I would also like to know the conditions that will warrant this status code.

Reply:
Thank you for your review @LukasChiama .  The 422 error is not returned in the controller but in the validation middleware. And since the validation middleware is part of the request-processing pipeline for that endpoint, then I think it's necessary the validation error response is documented as well.